rsconf/package_data/rsconf/nginx.conf.jinja

```diff
@@ -30,4 +30,12 @@ server {
         }
         root {{ rsconf.srv_d }};
     }
+
+    location /kickstart/ {
```

Comment:
We don't normally include the trailing slash.

Reply:
The ones above are regular expressions and we have the trick in dev of having the credentials in /srv so the names would collide with the host dirs.

Reply:
40b6ce1